financial-templates-lib/logger/SlackTransport.js

```diff
@@ -26,120 +26,122 @@ const SlackHook = require("winston-slack-webhook-transport");
 const { createEtherscanLinkMarkdown } = require("../../common/FormattingUtils");
 
 function slackFormatter(info) {
-  if (!("level" in info) || !("at" in info) || !("message" in info)) {
-    console.error("WINSTON INCORRECTLY CONFIGURED IN MESSAGE", info);
-    return {
+  try {
+    if (!("level" in info) || !("at" in info) || !("message" in info)) throw "WINSTON MESSAGE INCORRECTLY CONFIGURED";
```

Comment:
nit: should we use `new Error()` so we capture the stack trace?

Reply:
done :)